lms/djangoapps/dashboard/decisions/0002-deprecate-sysadmin-dashboard-adr

```diff
@@ -0,0 +1,45 @@
+1. Deprecating the Sysadmin Dashboard
+---------------------
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+Maintaining the sysadmin dashboard is challenging, and it is not widely used. The code is part of the lms
+application, even though most of it's use cases are relevant to course authoring.
+
+The sysadmin dashboard would be better suited as a pluggable django application, using appropriate APIs in the
+cms application
+
+Decision
```

Comment:
It seems the 2 remaining functionalities that are not replaceable by other features in the platform are:
* Creating a CMS account
* Importing a course from git

Curious: Do we need Python APIs for this or would exposing REST APIs be sufficient?

Reply:
REST APIs would be preferable, but I assumed that it would be bigger lift. Doesn't a REST API imply a python API?

Reply:
I think there actually may be a REST API for import now, but I don't know if it's considered a "public" interface. More importantly, I think it's missing a couple of key features like handling large courses (with lots of static assets) and sufficient error logging to help course authors understand why an import may have failed.